File: Python ejs/UD03/A02/UD03A025.py
import Math # type: ignore
n1=int(input("Introduzca un número: "))
n2=int(input("Introduzca otro número: "))
n3=int(input("Introduzca otro número: "))
if Math.abs(n3-n1)<Math.abs(n2-n3):
    print(f"{n3} está más cerca de  {n1}")
else:
    print(f"{n3} está más cerca de  {n2}")

File: Java ejs/UD3/A02/UD03A025.java
import java.util.Scanner;
public class UD03A025{
	public static void main(String[] args){
		int n1,n2,n3;
        Scanner teclado=new Scanner(System.in);
        System.out.println("Introduzca un número: ");
        n1=teclado.nextInt();
        System.out.println("Introduzca otro número: ");
        n2=teclado.nextInt();
        System.out.println("Introduzca otro número: ");
        n3=teclado.nextInt();
        if (Math.abs(n3-n1)<Math.abs(n2-n3))
        {
            System.out.println(n3+" está más cerca de "+ n1);
        }else{
            System.out.println(n3+" está más cerca de "+ n2);
        }
	}
}